### DISCUSSION TITLE
Download Checksums

### DISCUSSION THREAD

**Discussion by mrdrparker** [memtest86plus/memtest86plus#204] [Q&A] [answered]
Hello,
Are the checksums for the downloads listed on your website up to date?
Also, are they sha-256? sha-512? or something else.

Thanks!

**Comment by debrouxl** [comment, marked as answer]
The MD5 hashes provided on the website match the ones I've just obtained after downloading fresh copies of the files:

```
$ md5sum mt86plus* 
4a7758b4bcfa48177a3274fdb27a38f5  mt86plus_6.00_32.iso.zip6_64
cfe1ed0f55b31d447edefd841679d612  mt86plus_6.00_64.grub.iso.zip
cbbf4cc9245c0338b69ec6c274dd38c3  mt86plus_6.00_64.iso.zip
7e02984aa78f738689115cadd49a6cbc  mt86plus_6.00_USB_Installer.exe
```

On my side, the SHA-256 sums are:
```
$ sha256sum mt86plus*
43d866f95909a9d994ffd667c449919d5bd7337fc0bdaf2990b08ea30ee18869  mt86plus_6.00_32.iso.zip
e3263d1adf44a70ff6d317996b64656783ba8ccaa7094c7c032644bbd40dfc70  mt86plus_6.00_64.grub.iso.zip
5290e76e9a1930726c8fb249536924c23d66322355693a41c68f284ccd38456e  mt86plus_6.00_64.iso.zip
afb8f4bbe5da4ec862a60daee8dfd1a4b6938c46b568acec23601a6547ffbcbc  mt86plus_6.00_USB_Installer.exe
```

Sam, you should probably mention that the hashes currently displayed on memtest.org are MD5, and add the SHA-256 hashes, because MD5 is considered very weak nowadays :)

**Reply by mrdrparker** [reply]
Thanks! I completely forgot about md5, my bad.
